src/problems/Turbulence/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+if (AMReX_SPACEDIM EQUAL 3)
+    quokka_add_problem(JOB_NAME Turbulence ADD_TEST OFF)
```

Comment:
I think we need a fast-running version of this added to the tests so we can ensure it works when testing future PRs. This is separate from the nightly regression test version in `regression/quokka-tests.ini`. This version only needs to run for a few timesteps to ensure it doesn't immediately crash at runtime.